src/events/PickResidenceEvent.ts

```diff
@@ -0,0 +1,69 @@
+import { GamePlayMode, IEvent, IChoice, StatChanges } from "./core";
+
+export default class PickResidenceEvent implements IEvent {
+    prompt(): string {
+        return "It's time to move into your first year residence!";
+    }
+
+    imgPath(): string {
+        return "";
+    }
+
+    choices(): IChoice[] {
+        return [
+            new TotemPark(),
+            new PlaceVanier(),
+            new OrchardCommons()
+        ];
+    }
+
+    hasBottomBoxBorder(): boolean {
+        return false;
+    }
+
+    gamePlayMode(): GamePlayMode {
+        return GamePlayMode.Hide;
+    }
+}
+
+class TotemPark implements IChoice {
+    answer(): string {
+        return "Totem Park";
+    }
+
+    followUps(): IEvent[] {
+        return [];
+    }
+
+    statChanges(): StatChanges {
+        return new StatChanges(0, 0, 0);
+    }
+}
+
+class PlaceVanier implements IChoice {
+    answer(): string {
+        return "Place Vanier";
+    }
+
+    followUps(): IEvent[] {
+        return [];
+    }
+
+    statChanges(): StatChanges {
+        return new StatChanges(0, 0, 0);
+    }
+}
+
+class OrchardCommons implements IChoice {
+    answer(): string {
+        return "Orchard Commons";
+    }
+
+    followUps(): IEvent[] {
+        return [];
+    }
+
+    statChanges(): StatChanges {
+        return new StatChanges(0, 0, 0);
```

Comment:
So is this just a filler event that we will have to implement some actual effect in the game for at a later time? Notice they don't have any stat changes.

Reply:
no it won't change stats, its just an event to add storyline to make it feel like an actual simulation